drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -895,6 +896,48 @@ func GetContainerPVCMountMap(pod corev1.Pod) map[string][]string {
 	return containerPaths
 }
 
+// GetContainerPVCMountMapWithSC fetches storage class along with mount paths for containers using PVCs
+func GetContainerPVCMountMapWithSC(pod corev1.Pod) (map[string]*storagev1.StorageClass, error) {
```

Comment:
Can we try hard and soft symbolic links?

Reply:
Thanks for suggestion @kshithijiyer-px , Added the logic and a step to create file with symlink and hardlink in CreateFilesInPodDirectory

-rw-r--r--  2 root     root       53 Dec  8 09:43 linkFile.txt.hardlink
lrwxrwxrwx  1 root     root       37 Dec  8 09:43 linkFile.txt.symlink -> /var/lib/postgresql/data/linkFile.txt